lld/ELF/Arch/LoongArch.cpp

```diff
@@ -63,6 +76,7 @@ enum Reg {
   R_ZERO = 0,
   R_RA = 1,
   R_TP = 2,
+  R_A0 = 4,
```

Comment:
> Seems it is not used by current patch.

Yes, thanks for remainding.
Because the code of this pr is split from a complete relaxation patch, there may be partially unused code. I will check the code of this pr again as soon as possible.